.prettierrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "singleQuote": true,
+  "tabWidth": 2,
```

Comment:
This is already the default I believe?

Reply:
Yes but for some reason it didn't work without the option explicitly set. I think it might be because of the `.editorconfig` option (which I later changed).

Reply:
Tested and fixed